### PR TITLE
#97 - Add validateOptions implementations

### PR DESCRIPTION
**Pull request by SimonMellerin**
Tests are comming

### CODE REVIEW

File: src/Anonymization/Anonymizer/AbstractMultipleColumnAnonymizer.php [hunk 1]
```diff
@@ -45,6 +45,28 @@ protected function getColumnTypes(): array
...
+    #[\Override]
+    protected function validateOptions(): void
+    {
+        $options = $this->options->all();
+
+        if (0 === \count($options)) {
+            throw new \InvalidArgumentException("You must provide at least one option.");
+        }
+
+        if (\count(\array_unique($options)) < \count($options)) {
```
Comment (pounard) [inline src/Anonymization/Anonymizer/AbstractMultipleColumnAnonymizer.php:57]: What happens if there are options different from colum names ?

File: src/Anonymization/Anonymizer/Core/ConstantAnonymizer.php [hunk 2]
```diff
@@ -21,12 +21,14 @@
 class ConstantAnonymizer extends AbstractAnonymizer
 {
     #[\Override]
-    public function anonymize(Update $update): void
+    protected function validateOptions(): void
     {
-        if (!$this->options->has('value')) {
-            throw new \InvalidArgumentException("'value' option should be given.");
-        }
+        $this->options->get('value', null, true);
```
Comment (pounard) [inline src/Anonymization/Anonymizer/Core/ConstantAnonymizer.php:26]: Basic, simple.

File: src/Anonymization/Anonymizer/AbstractMultipleColumnAnonymizer.php (hunk 1 above)
Comment (SimonMellerin) [reply src/Anonymization/Anonymizer/AbstractMultipleColumnAnonymizer.php:57]: Good point!

Comment (SimonMellerin) [reply src/Anonymization/Anonymizer/AbstractMultipleColumnAnonymizer.php:63]: done

**Review by pounard — APPROVED** [review]

**Pull request merged**